proposals/wac-ucr/invitation.ttl

```diff
@@ -0,0 +1,70 @@
+@prefix eco: <http://www.w3.org/ns/solid/ecosystem#>
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#>
+
+<#invitation>
+    a eco:AccessInvitation ;
+    eco:registeredBy https://alice.pod.example/profile/id#me ;
+    eco:registeredWith https://trusted.example/id#agent ;
+    eco:registeredAt "2020-09-13T19:32:52Z"^^xsd:dateTime ;
+    eco:updatedAt "2020-09-13T19:32:52Z"^^xsd:dateTime ;
+    eco:expiresAt "2020-09-20T19:32:52Z"^^xsd:dateTime ;
+    eco:hasPreparedAccessNeedGroup <#pag-notebook-group> ;
+    eco:hasDataGrant <#notebook-grant>, <#note-grant> ;
+    eco:hasAccessInvitationChannel <#phone-channel>, <#email-channel> .
+
+  <#phone-channel>
+    a eco:PhoneInvitationChannel ;
+    eco:channelTarget "(555) 555-1212" ;
+    eco:channelCode "654345" ;
+    eco:isValidated false ;
+    eco:remainingAttempts 3 .
+
+  <#email-channel>
+    a eco:EmailInvitationChannel ;
+    eco:channelTarget "bob@example.com" ;
+    eco:channelCode "434567" ;
+    eco:isValidated true ;
+    eco:remainingAttempts 3 .
+
+  <#notebook-grant>
+    a eco:DataGrant ;
+    eco:hasRegistration data:notebook-tree ;
+    eco:registeredShapeTree note:notebook-tree ;
+    eco:satisfiesPreparedAccessNeed <#pan-notebook> ;
+    eco:accessMode acl:Read, acl:Write ;
+    eco:scopeOfDataGrant eco:AllInstances ;
+
+  <#note-grant>
+    a eco:DataGrant ;
+    eco:hasRegistration data:note-tree ;
+    eco:registeredShapeTree note:note-tree ;
+    eco:satisfiesPreparedAccessNeed <#pan-note> ;
+    eco:accessMode acl:Read, acl:Write ;
+    eco:scopeOfDataGrant eco:AllInstances ;
+
+  <#pag-notebook-group>
+    a eco:PreparedAccessNeedGroup ;
+    eco:fromAccessNeedGroup ex:accessNotebookGroup ;
+    eco:hasAccessNeedGroupDecorator ex:accessNotebookGroupDecorator ;
+    eco:accessNecessity eco:AccessRequired ;
+    eco:authenticatesAs eco:Pilot ;
+    eco:hasPreparedAccessNeed <#pan-notebook> .
+
+  <#pan-notebook>
+      a eco:PreparedAccessNeed ;
+      eco:registeredShapeTree note:notebook-tree ;
+      eco:accessMode acl:Read, acl:Write ;
+      eco:accessNecessity eco:AccessRequired ;
+      eco:hasAccessNeedDecorator ex:notebookAccessDecorator ;
+      eco:hasShapeTreeDecorator note:notebookDecorator ;
+      eco:referencesPreparedAccessNeed <#pan-notes> ;
+      eco:hasDataGrant <#notebook-grant> .
+
+  <#pan-note>
+      a eco:PreparedAccessNeed ;
+      eco:registeredShapeTree note:note-tree ;
+      eco:accessMode acl:Read, acl:Write ;
+      eco:accessNecessity eco:AccessRequired ;
+      eco:hasAccessNeedDecorator ex:noteAccessDecorator ;
+      eco:hasShapeTreeDecorator note:noteDecorator ;
+      eco:hasDataGrant <#note-grant> .
```

Comment:
Added this file by mistake?

Reply:
Ah - yes late night commits! thanks 🙂

Reply:
Cleared in https://github.com/solid/authorization-panel/pull/152/commits/01be17329981132f3454e3f0ff542c287113d687